### ISSUE TITLE
Package jfrog-cli-2.61.0.x86_64.rpm is not signed

### ISSUE THREAD

**Issue by juliagomezi** [jfrog/jfrog-cli#2620]
### Describe the bug

Trying to install the jfrog-cli RPM version 2.61.0 in a RHEL 8 fails due to the RPM not being signed:
```
+ echo '[jfrog-cli]' > jfrog-cli.repo
+ echo name=jfrog-cli >> jfrog-cli.repo
+ echo baseurl=https://releases.jfrog.io/artifactory/jfrog-rpms >> jfrog-cli.repo
+ echo enabled=1 >> jfrog-cli.repo
+ sudo rpm --import https://releases.jfrog.io/artifactory/jfrog-gpg-public/jfrog_public_gpg.key
+ sudo mv jfrog-cli.repo /etc/yum.repos.d/
+ sudo dnf install --assumeyes jfrog-cli-v2-jf
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered with an entitlement server. You can use subscription-manager to register.

jfrog-cli                                       613 kB/s | 255 kB     00:00    
Package jfrog-cli-v2-jf-2.59.1-1.x86_64 is already installed.
Dependencies resolved.
================================================================================
 Package                Architecture  Version            Repository        Size
================================================================================
Upgrading:
 jfrog-cli-v2-jf        x86_64        2.61.0-1           jfrog-cli        9.0 M

Transaction Summary
================================================================================
Upgrade  1 Package

Total download size: 9.0 M
Downloading Packages:
jfrog-cli-2.61.0.x86_64.rpm                      13 MB/s | 9.0 MB     00:00    
--------------------------------------------------------------------------------
Total                                            13 MB/s | 9.0 MB     00:00     
Package jfrog-cli-2.61.0.x86_64.rpm is not signed
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED`
```
I went back to previous versions and the first one I found that is signed is version 2.59.1.

### Current behavior

Cannot install the jfrog-cli RPM 2.61.0 version in a RHEL8 due to it not being signed.

### Reproduction steps

Try to install jfrog-cli-2.61.0.x86_64 RPM in a RHEL 8.

### JFrog CLI version

2.61.0

### Operating system type and version

RHEL 8

**Comment by eyalbe4** [comment]
@juliagomezi,
JFrog CLI 2.61.1 is out and includes the fix.
Please accept our apologies for the inconvenience caused by this.

**Comment by mathewhoopertr** [comment]
@eyalbe4 This needs to be reopened - 

```
dnf install -y jfrog-cli-v2-jf 
Repository packages-microsoft-com-prod is listed more than once in the configuration
Last metadata expiration check: 1:23:27 ago on Fri 26 Jul 2024 06:12:08 AM CDT.
Package jfrog-cli-v2-jf-2.59.1-1.x86_64 is already installed.
Dependencies resolved.
=========================================================================================================================================================================================
 Package                                           Architecture                             Version                                    Repository                                   Size
=========================================================================================================================================================================================
Upgrading:
 jfrog-cli-v2-jf                                   x86_64                                   2.61.1-1                                   jfrog-cli                                   9.0 M

Transaction Summary
=========================================================================================================================================================================================
Upgrade  1 Package

Total size: 9.0 M
Downloading Packages:
[SKIPPED] jfrog-cli-2.61.1.x86_64.rpm: Already downloaded                                                                                                                               
Package jfrog-cli-2.61.1.x86_64.rpm is not signed
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED
```

**Comment by eyalbe4** [comment]
We're looking into this @mathewhoopertr.

**Comment by eyalbe4** [comment]
@mathewhoopertr,
Let us know if the newly released version 2.61.2 resolves the issue for you.

**Comment by sverdlov93** [comment]
> @mathewhoopertr, Let us know if the newly released version 2.61.2 resolves the issue for you.
@afarresg, @davidmateos, @albertcampana, @spira0, @apanza2, @kumadee, @juliagomezi

**Comment by kumadee** [comment]
Yes, I can confirm that the issue is fixed with last released version. Thanks

**Comment by davidmateos** [comment]
I can also confirm that the issue is fixed in 2.61.2
